AdyenComponents/BACS Direct Debit/Trackers/BACSDirectDebitComponentTracker.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+/// :nodoc:
+internal protocol BACSDirectDebitComponentTrackerProtocol: AnyObject {
```

Comment:
This looks like something we can use more often 🤔
How about make it `ComponentTrackerProtocol` and move to **Core** ?